arcs_fact_tests/ok_claim_propagates.dl

```diff
@@ -0,0 +1,32 @@
+// claim propagates through a chain of particles
+// #Ingress: P1
+// #OK
+//particle P1
+//  foo: writes Foo {}
+//  claim foo is trusted
```

Comment:
@markww This comment is not pertinent to this PR, but as you observed `trusted` is not the correct tag. We should probably call it `secret` or something like that.

Reply:
Acknowledged, I'll resolve this comment once I file that bug I promised you.

Reply:
See issue #34